style.css

```diff
@@ -7,6 +7,10 @@ body {
 	padding:0;
 	background-color:white;
 	font: 14px 'Helvetica Neue','Nimbus Sans','Helvetica','Arial',sans-serif;
+	-webkit-box-orient:vertical;
```

Comment:
How did the linter not complain about this?

Reply:
Linter? The wikipedia.de repository is anarchy.